config/appmodel/moduleconfs.data.xml

```diff
@@ -273,4 +296,46 @@
  <attr name="prescale" type="u32" val="1000"/>
 </obj>
 
+<obj class="WIBConf" id="wib1001-conf">
```

Comment:
These OKS objects (WIBConf) should not be in the generic moduleconfs file, but, either in a separate file or in the CRP4 session file.